decompiler/backend/codevisitor.py

```diff
@@ -205,7 +205,11 @@ def _condition_string(self, condition: ConditionVar) -> str:
             return f"{condition}"
         condition = condition.simplify_to_shortest(self.complexity_bound)
 
-        if condition.is_symbol:
+        if condition.is_true:
+            return "true"
+        elif condition.is_false:
+            return "false"
+        elif condition.is_symbol:
```

Comment:
Could you add a test? This way we hopefully do not remove it at some point accidentally.

Reply:
I added a simple test to `test_codegenerator.py` containing the condition `true`. The test fails on the current master and succeeds on this PR - hope that is sufficient.